src/actor.rs

```diff
@@ -54,6 +101,18 @@ impl PartialEq for Sig {
     }
 }
 
+impl PartialOrd for Sig {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        self.0.to_bytes().partial_cmp(&other.0.to_bytes())
+    }
+}
+
+impl Ord for Sig {
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.partial_cmp(&other).unwrap()
```

Comment:
Perhaps consider no unwrap (we prohibit in prod code). Here is tricky though, but perhaps even desugar and log a Critical error? Possibly better to not rely on this Ord requirement. Maybe consider https://github.com/orlp/slotmap or similar?

Reply:
yes @davidrusu in general we return Result/errors or otherwise avoid/prohibit use of unwrap() and panic().  Something like unwrap_or_default() is ok when it makes sense as it will not panic.

I think @dirvine would agree it is worth taking an extra day or two of your time to refactor as necessary to avoid unwrap(), so that all cases are somehow handled at each step.

I understand it is faster/easier to prototype with unwrap(), but now that the code appears to be complete and working this appears a good time to clean these up.

Team policy also discourages use of unwrap() and panic() in test cases.  However, I think that in the interest of time/schedule, we can leave the tests as-is for now.

Reply:
I found a safe way to perform the comparison, turns out the underlying byte slices are Ord so we can simply call `cmp` on the the byte slices.

I've gone through and removed all other occurrences of panics/asserts/unwraps.